packages/database/src/migrations/20210610005659-DeleteCovid19FJFacilityOverlays-modifies-data.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const deleteOverlay = async (db, id) => {
+  return db.runSql(`
+    delete from "mapOverlay" where "id" = '${id}';
+    delete from "map_overlay_group_relation" where "child_id" = '${id}';
+  `);
+};
+
+exports.up = async function (db) {
+  await deleteOverlay(db, 'FJ_COVID_TRACKING_Dose_1_Facility');
+  await deleteOverlay(db, 'FJ_COVID_TRACKING_Dose_2_Facility');
```

Comment:
These overlays don't make sense anymore because data level is now at sub_district

Reply:
From [#2929](https://github.com/beyondessential/tupaia-backlog/issues/2929) I think we want to archive them, instead of deleting?

And we need to clean up dashboard `FJ_COVID_TRACKING_By_Gender_Country` as well.